### DISCUSSION TITLE
Provision public dashboards

### DISCUSSION THREAD

**Discussion by EdwardKuenen** [grafana/grafana#71702] [Dashboards, Panels & Visualizations]
It seems to me that public dashboards can not be provisioned, but must be a manual and cumbersome process.

I would very like to provision public dashboards, just like normal dashboards. The path of the public dashboard should not be random, but a specified path to use the same path in more environments, like production and pre-production.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #83017. Feel free to continue the discussion around this there. Thank you!
